package.json

```diff
@@ -140,7 +140,8 @@
       "src/**/*.cy.js",
       "src/{App,entry,entry-dev}.js",
       "**/__fixtures__/**",
-      "**/__factories__/**"
+      "**/__factories__/**",
+      "src/Components/RegistrationProcess/CreateActivationKeyModal.js"
```

Comment:
We shouldn't make exceptions like this. It feels a little bit like cheating. It's ok, if this appears as uncovered, and if we really don't wanna see any red lines, we can mock the AsyncComponent and still call and render the `CreateActivationKeyModal` component and it'll be "covered"

Reply:
note: it doesn't